src/dataprep/main/load_proquest/magfos_to_db.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# %%
+"""
+Read in mag field of study of dissertation abstracts (estimated on TDM studio) and save to database.
+
+Create table in database:
+- pq_magfos with the estimated fields of study
+
+pq_magfos data schema is:
+
+goid, fieldrank INTEGER, FieldOfStudyId INTEGER, score NUMERIC 
+
+unique index on goid and FieldOfStudyId (multiple fields per goid)
```

Comment:
can you quickly describe what is fieldrank?

Reply:
done